tests/mito.lib/io/read_mesh_summit_segment_3D.cc

```diff
@@ -4,41 +4,34 @@
 //
 
 #include <gtest/gtest.h>
-#include <mito/mito.h>
+#include <mito/base.h>
+#include <mito/io.h>
+#include <mito/mesh.h>
 
 
 TEST(SummitReader, LoadSummitSegmentsMesh3D)
 {
-    // an empty topology
-    auto & topology = mito::topology::topology();
-
-    // an empty cloud of points
-    auto & point_cloud = mito::geometry::point_cloud<3>();
-
-    // a 3D geometry binding the topology {topology} on the cloud of points {point_cloud}
-    auto & geometry = mito::geometry::geometry(topology, point_cloud);
-
     // a Euclidean coordinate system in 3D
     auto coord_system = mito::geometry::coordinate_system<3, mito::geometry::EUCLIDEAN>();
 
     std::ifstream fileStream("segment.summit");
-    auto mesh =
-        mito::io::summit::reader<mito::topology::segment_t>(fileStream, geometry, coord_system);
+    auto mesh = mito::io::summit::reader<mito::geometry::segment_t<3>>(fileStream, coord_system);
 
     // assert you read 10 cells
     EXPECT_EQ(mesh.nCells(), 10);
 
-    // assert you found 2 cells (vertices) on the boundary
+    // assert you found 2 nodes on the boundary
     {
         auto boundary_mesh = mesh.boundary();
         EXPECT_EQ(boundary_mesh.nCells(), 2);
     }
 
+#if 0    // TOFIX
```

Comment:
Can this be removed? Do we need a ticket?

Reply:
This is fixed in PR #64.